ajax/libs/winstrap/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "winstrap",
+  "filename": "css/winstrap.min.css",
+  "description": "Winstrap is the official Bootstrap theme for Microsoft design language.",
+  "version": "0.5.11",
+  "author": "Microsoft",
+  "licenses": [
+    "MIT",
+    "OFL-1.1"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/winjs/winstrap.git"
+  },
+  "keywords": [
+    "microsoft",
+    "windows",
+    "bootstrap",
+    "theme"
+  ],
+  "homepage": "http://winstrap.azurewebsites.net",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/winjs/winstrap.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
Let's use `+(css|fonts)/**/!(*.txt)` to exclude `LICENSE.txt` & `js/vendor/*` because the files under `js/vendor` can be used from  https://cdnjs.com/libraries/twitter-bootstrap and https://cdnjs.com/libraries/jquery, and `LICENSE.txt` can't be added since it's not a necessary front-end file.
Thank you.

Memo: I want to use excluded mode in `dist` but it's hard to me to design the pattern. even I use `!(vendor){,**/!(*.txt)}` I can't grab `fonts/glyphs/*` files. If someone could help it, it's welcome.